index.html

```diff
@@ -1,11 +1,130 @@
 <!DOCTYPE html>
 <html>
     <head>
+        <meta charset="UTF-8">
+        <meta name="author" content="Thomas">
         <title>Thomas</title>
     </head>
     <body>
-        <h1>Hi everyone! This is my first website!</h1>
-        <h4>Well, actually my second!</h4>
-        <p>My first one was using cursor!</p>
+        <h1>Welcome! I'm Thomas.</h1>
+        <p>
+           I am a <em>4th year</em> student at <strong>Bishop's University</strong>
+           I am from Ottawa, Ontario. <br>
+           And I play on the lacrosse team here at school.
+        </p>
+        <hr>
+        <h2><b>Facts about me:</b></h2>
+        <ul>
+            <li>DOB: July 28, 2004</li>
+            <li>Hometown: Ottawa</li>
+            <li>Program: Business Technology &amp Analytics</li>
+            <li>Graduation Year: 2026</li>
+        </ul>
+        <h2>Competitive Sports Career</h2>
+        <table border="1" style="width:100%">
+            <thead>
+                <tr>
+                    <th>Year</th>
+                    <th>Hockey</th>
+                    <th>Lacrosse</th>
+                </tr>
```

Comment:
The `</thead>` should be here, and then start a `<tbody>` below. This way, you can eventually style the first line differently from the rest.